Tests/youdaTests/IAPServiceMockTest.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  youda
+//
+//  Copyright (c) 2020 Industrial Binaries
+//  MIT license, see LICENSE file for details
+//
+
+import StoreKit
+import XCTest
+@testable import youda
+
+final class IAPServiceMockTest: XCTestCase {
+  func testAvailableProducts() {
+    let apiService = IAPServiceMock(
+      availableProducts: .mockAvailableProducts,
+      purchasedProducts: []
+    )
+
+    XCTAssertEqual(apiService.availableProducts.first?.productIdentifier, .purchasedAvailableIdentifier)
```

Comment:
If we expect just one product, we should also add tests for `apiService.availableProducts.count` == 1